utils/req-res-log-validator.py

```diff
@@ -43,7 +43,7 @@
 1. Fail the script if one or more of the branches of the reply schema (e.g. oneOf, anyOf) was not hit.
 """
 
-IGNORED_COMMANDS = [
+IGNORED_COMMANDS = {
```

Comment:
any specific reason to convert this from an array to a set?
is it just an optimization (faster lookup)?

Reply:
yean, for a faster lookup, if we can handle the sentinel info-cache, i think we can drop it (undo the change if needed)

Reply:
it's logically better, let's keep it.